core/templates/dev/head/css/oppia.css

```diff
@@ -1026,8 +1089,8 @@ md-card.oppia-gallery-tile {
 }
 
 .oppia-html-select .oppia-html-select-option oppia-noninteractive-image {
```

Comment:
OK, here is another way to look at it.

The view [here](https://github.com/oppia/oppia/pull/1638/files) shows the changes your PR makes to oppia:develop. If you don't mean to make changes to classes like .oppia-html-select or similar, then there should be no change shown there. (But there is, so there's a problem.)

That is to say, when you've done it correctly, that diff view should only reflect the changes you've deliberately made, and nothing else.